contrib/vectorscan/template.py

```diff
@@ -0,0 +1,40 @@
+pkgname = "vectorscan"
+pkgver = "5.4.11"
+pkgrel = 0
+archs = ["aarch64", "ppc64", "x86_64"]
+build_style = "cmake"
+configure_args = [
+    "-DCMAKE_BUILD_TYPE=Release",
+    "-DCMAKE_SKIP_INSTALL_RPATH=ON",
+    "-DBUILD_SHARED_LIBS=ON",
+    "-DFAT_RUNTIME=OFF",
+]
+hostmakedepends = ["cmake", "ninja", "pkgconf", "ragel"]
+makedepends = ["boost-devel", "sqlite-devel"]
+pkgdesc = "High-performance regular expression matching library"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "BSD-3-Clause"
+url = "https://www.vectorcamp.gr/vectorscan/index.html"
```

Comment:
Link redirects to https://www.vectorcamp.gr/ which doesn't seem to describe the library.

Reply:
Okay will find the correct one

Reply:
without `index.html` it works, tried `home` and `default` with html and htm extensions as well and all redirected

Reply:
it works without a `/` on the end of vectorscan as well